model/ExpandedLicensing/Individuals/NoAssertionLicense.md

```diff
@@ -15,7 +15,7 @@ the SPDX creator has intentionally provided no information (no meaning should be
 ## Metadata
 
 - name: NoAssertionLicense
-- type: /SimpleLicensing/AnyLicenseInfo
+- type: IndividualLicensingInfo
```

Comment:
why is [model/ExpandedLicensing/Individuals/NoAssertionLicense.md](https://github.com/spdx/spdx-3-model/pull/588/files#diff-7f1aaba6ef962a460d62c1e2e96b9381dbd86a6a1d0ff06ce26f16966a579b16) not of type [model/ExpandedLicensing/Classes/NoAssertionLicense.md](https://github.com/spdx/spdx-3-model/pull/588/files#diff-2ac21c661dbfd86dbf49893a827f500545da2efb57b35a664a69cb3ba09ac2b7)? I am confused to see it in Individuals and Classes an the same time